packages/Uploader/stories/Uploader.stories.js

```diff
@@ -0,0 +1,115 @@
+import React from "react";
+import { storiesOf } from "@storybook/react";
+import Uploader from "../src/Uploader";
+import Testing from "./Testing";
+
+storiesOf("Uploader", module)
+  .add("Basic example with all files successfully uploaded", () => (
+    <>
+      <Uploader
+        endpoint="http://localhost:9000/upload.php"
+        onChange={files => {
+          console.log("onChange files:", files);
```

Comment:
I'd suggest using the `actions` addon instead of a console log.

Reply:
I will polish stories and everything in my next PR, still need to build the UI, this is just logic handle. :)